doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,348 @@
+__children["RaySwitchShader"]["parameters"]["camera"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["shadow"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["diffuse_reflection"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["diffuse_transmission"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["specular_reflection"].setInput( __children["TraceSet1"]["out"] )
+__children["RaySwitchShader"]["parameters"]["specular_transmission"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["volume"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["__uiPosition"].setValue( imath.V2f( -9.90797043, -103.52475 ) )
+__children["ShaderAssignment"]["enabled"].setValue( False )
+__children["ShaderAssignment"]["in"].setInput( __children["ShaderAssignment1"]["out"] )
+__children["ShaderAssignment"]["filter"].setInput( __children["PathFilter"]["out"] )
+__children["ShaderAssignment"]["shader"].setInput( __children["RaySwitchShader"]["out"] )
+__children["ShaderAssignment"]["__contextCompatibility"].setValue( False )
+__children["ShaderAssignment"]["__uiPosition"].setValue( imath.V2f( 7.23167801, -103.52475 ) )
+__children["PathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/world/geo/plane' ] ) )
+__children["PathFilter"]["__uiPosition"].setValue( imath.V2f( 18.0980415, -97.9078064 ) )
+__children["StandardSurface"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface"]["__uiPosition"].setValue( imath.V2f( -39.750576, -99.9247513 ) )
+__children["StandardSurface2"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface2"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface2"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface2"]["__uiPosition"].setValue( imath.V2f( -39.509861, -115.691742 ) )
+__children["TraceSet1"]["parameters"]["passthrough"].setInput( __children["StandardSurface2"]["out"] )
+__children["TraceSet1"]["parameters"]["trace_set"].setValue( 'spheres' )
+__children["TraceSet1"]["parameters"]["inclusive"].setValue( False )
+__children["TraceSet1"]["__uiPosition"].setValue( imath.V2f( -24.7410622, -115.691742 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop4"], 'nodeGadget:color', imath.Color3f( 0.347900003, 0.438600004, 0.5 ) )
+__children["Backdrop4"]["title"].setValue( 'Note: New objects' )
+__children["Backdrop4"]["description"].setValue( 'Nodes such as Sphere, Plane, lights, etc have a Sets plug. This can be used to put them in one or more sets at creation without needing the separate Set node.' )
```

Comment:
### Backdrop text
~~Nodes~~ **Location-generating scene nodes,** such as Sphere, Plane, **Camera**, lights, etc have a Sets plug. ~~This can be used to put them in one or more sets at creation without needing the separate Set node.~~ **You can assign them sets by inputting a space-separated list of set names into this plug, instead of a connecting them to a separate Set node.**

Reply:
I updated this a little so it makes it clear that the sets are assigned to the locations, not the nodes. I also used 'creating' not 'generating' as generating generally covered more procedural nodes too, many of which don't have Sets plugs.

> Many location-creating scene nodes such as Sphere, Plane, Camera, lights, etc have a Sets plug.
>
> You can assign sets to the new locations by inputting a space-separated list of set names into this plug, instead needing separate Set node(s) downstream.